docker/Dockerfile.faux

```diff
@@ -20,16 +20,32 @@ RUN pubber/bin/build
 
 FROM daq/aardvark:latest
 
+#SSH ENV Settings
+ENV DEBIAN_FRONTEND noninteractive
+RUN echo exit 0 > /usr/sbin/policy-rc.d
+
 # Run this separately so it can be shared with other builds.
 RUN $AG update && $AG install openjdk-8-jre
 
 RUN $AG update && $AG install isc-dhcp-client ethtool network-manager netcat curl\
-    python ifupdown openssl ssh
+    python ifupdown openssl ssh openssh-server
 
 # Prefetch resolvconf to dynamically install at runtime in start_faux.
 RUN $AG update && cd /tmp && ln -s ~/bin bin && $AG download resolvconf && mv resolvconf_*.deb ~
+RUN $AG update && $AG install -y telnetd
+RUN $AG update && $AG install xinetd
```

Comment:
Are you using telnet at all in this test?

Reply:
I'm unsure how we manage this, going forward this test will support telnet. Indeed it does support telnet right now but the faux device is particulary tricky to create.

If we remove these lines of code I feel that the valauble information will be lost into the ether

Reply:
The telnet lines have been removed, but I have a telnet branch on my own fork of daq so that the code is not lost in the ether

Reply:
@ktredstone update: we _definitely_ need telnet support in this test